ProcessMaker/processmaker
# QA observation bugfix/FOUR-12799: Select lists are different controls in the launch pad modal

## How to Test
-  Open Any Process with "Open in Launchpad"
- Check both Dropdowns/Select List sizes

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-12799

## Code Review Checklist
ci:deploy
ci:next
